### PR TITLE
bulk css changes to use https

### PR DESCRIPTION
**Pull request by PeterConstable**
Bulk search/replace changes are another situation in which a PR makes good sense.

### CODE REVIEW

**Comment by Ken-Whistler** [comment]
O.k., sure. You can make bulk changes this way. Could be useful in some cases, but they depend on the source being clean enough to apply bulk changes in the first place. And so now you make all these changes -- are you forcing every editor to interact directly with GitHub to pick up the changes, before they can continue work? Not all of the reports are *maintained* in SVN (now GitHub, if we get this working) -- for some, the repository is a periodic drop-off for documents at the publication milestone for a new version.

**Comment by PeterConstable** [comment]
Fair enough: if the contents in this repo aren't the current "truth" or "most recent truth all new commits should be based on", then we can ignore/close this proposed change.

**Comment by Ken-Whistler** [comment]
More on this in email. l0l

File: tr18/tr18.html [hunk 1]
```diff
@@ -6,7 +6,7 @@
 <head>
 <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
 <meta http-equiv="Content-Language" content="en-us">
-<link rel="stylesheet" type="text/css" href="../reports-v2.css">
+<link rel="stylesheet" type="text/css" href="https://www.unicode.org/reports/reports-v2.css">
```
Comment (markusicu) [inline tr18/tr18.html:9]: are relative paths no good?

Comment (PeterConstable) [reply tr18/tr18.html:9]: I would presume not when viewing a page located at github.com/unicode-org or at unicode-org.github.io

**Comment by PeterConstable** [comment]
Files were in different states and changes had to be done separately. Closing.

**Pull request closed without merging**